src/as-cache.c

```diff
@@ -829,6 +832,16 @@ as_cache_open (AsCache *cache, const gchar *fname, const gchar *locale, GError *
 		goto fail;
 	}
 
+	/* set FD_CLOEXEC manually. LMDB should do that, but it doesn't:
+	   https://www.openldap.org/lists/openldap-bugs/201702/msg00003.html */
+	rc = mdb_env_get_fd (priv->db_env, &db_fd);
+	if (rc == MDB_SUCCESS) {
+		db_fd_flags = fcntl (db_fd, F_GETFD);
+	}
+	if (db_fd_flags != -1) {
+		fcntl (db_fd, F_SETFD, db_fd_flags | FD_CLOEXEC);
+	}
```

Comment:
Could you remove the extra brackets? If there's just a single line following an if statement, they should be omitted in AppStream's coding style